### DISCUSSION TITLE
Failed to use remote supercomputing resources

### DISCUSSION THREAD

**Discussion by Saberve** [deepmodeling/dpgen#931] [Q&A]
When I run dpgen init_surf on the local server and submit the task to the remote supercomputer, the computing task is terminated。

![image](https://user-images.githubusercontent.com/53855907/187725530-d48c8062-90d2-4c77-818c-fcea967ff932.png)

The relevant modules have been installed on the supercomputer. But when I add the module load command to the machine.json file, the error message is repeated。

![image](https://user-images.githubusercontent.com/53855907/187726036-b9a92260-aef4-4b7a-9c7b-3337b037fde7.png)

**Comment by HuangJiameng** [comment]
<img width="322" alt="image" src="https://user-images.githubusercontent.com/105633685/187823667-380dea38-8743-4e1e-b927-5e30abc2bfbb.png">

Maybe it is the reason of those errors?

**Reply by Saberve** [reply]
Thank you for your answer。But I checked the ID of the supercomputing user used and it is correct.
And the error message looks like looking for user and these modules on my local server.
it's confusing.

**Reply by HuangJiameng** [reply]
@KZHIWEI Could you please take a look? THX!

**Reply by AnguseZhang** [reply]
@Saberve This problem is not directly to DP-GEN.  You should check the slurm scripts and try to find how to successfully run a job on your cluster manually, not via DP-GEN. And after you work out,  you can modify the `machine.json` according to the experience.

**Reply by AnguseZhang** [reply]
One tip is that some clusters forbid the users to submit job from the computing node, and neither does `DP-GEN` require.
